help/destinations/catalog/data-partners/merkury-enterprise-identity.md

```diff
@@ -0,0 +1,168 @@
+---
+keywords: Experience Platform;home;popular topics;destinations;connectors;destination connectors;destinations sdk;sdk;SDK
+title: Merkury Enterprise Connections Documentation self-service template for the UI
+description: Learn how to create a Merkury Enterprise Identity destionation connection using the Adobe Experience Platform UI.
+---
+
+# Merkury Enterprise Identity Destination
+
+## Overview
+
+Clients of Adobe Experience Platform use the Merkury Enterprise Identity Destination to build more accurate, comprehensive, and insightful consumer profiles. With improved profile data, Marketers can power better insights, segments, and models resulting in more accurate targeting and predictive modeling.
+
+![A diagram of a customer data platform Description automatically
+generated](../../assets/catalog/data-partners/merkury-identity/media/image1.png)
+
+Follow the steps in this documentation page to create a Merkury Connections destination connection and activate data using the Adobe Experience Platform user interface.
+
+![A diagram showing the interconnection between Merkury and Experience Platform, including ingestion and activation](../../assets/catalog/data-partners/merkury-identity/media/image2.png)
+
```

Comment:
can we also add a brief note or paragraph about how this destination is different to the other Merkury destination and when to use each of them?

Reply:
I have updated this to have a note about the other connector.